apps/heft/includes/jest-shared.config.json

```diff
@@ -3,9 +3,11 @@
 
   "rootDir": "../../../../",
   "testURL": "http://localhost/",
-  "testMatch": ["<rootDir>/src/**/*.test.ts?(x)"],
+  "testMatch": ["<rootDir>/src/**/*.test.{ts,tsx}"],
+  "transformIgnorePatterns": [],
```

Comment:
Note: The default `"transformIgnorePatterns"` trims anything with `node_modules` in the path, which is not what we want for .css files